packages/suite/src/views/onboarding/steps/SecurityCheck/SecurityCheckFail.tsx

```diff
@@ -64,28 +64,33 @@ interface SecurityCheckFailProps {
     goBack?: () => void;
 }
 
-export const SecurityCheckFail = ({ goBack }: SecurityCheckFailProps) => (
-    <SecurityCheckLayout isFailed>
-        <TopSection>
-            <StyledH1>
-                <Translation id="TR_DEVICE_COMPROMISED_HEADING" />
-            </StyledH1>
-            <Text>
-                <Translation id="TR_DEVICE_COMPROMISED_TEXT" />
-            </Text>
-        </TopSection>
-        <SecurityChecklist items={checklistItems} />
-        <Buttons>
-            {goBack && (
-                <SecurityCheckButton variant="secondary" onClick={goBack}>
-                    <Translation id="TR_BACK" />
-                </SecurityCheckButton>
-            )}
-            <StyledTrezorLink variant="nostyle" href={supportChatUrl}>
-                <StyledSecurityCheckButton>
-                    <Translation id="TR_CONTACT_TREZOR_SUPPORT" />
-                </StyledSecurityCheckButton>
-            </StyledTrezorLink>
-        </Buttons>
-    </SecurityCheckLayout>
-);
+export const SecurityCheckFail = ({ goBack }: SecurityCheckFailProps) => {
+    const heading = goBack ? 'TR_DEVICE_COMPROMISED_HEADING_SOFT' : 'TR_DEVICE_COMPROMISED_HEADING';
+    const text = goBack ? 'TR_DEVICE_COMPROMISED_TEXT_SOFT' : 'TR_DEVICE_COMPROMISED_TEXT';
+
+    return (
+        <SecurityCheckLayout isFailed>
+            <TopSection>
+                <StyledH1>
+                    <Translation id={heading} />
+                </StyledH1>
+                <Text>
+                    <Translation id={text} />
+                </Text>
+            </TopSection>
+            <SecurityChecklist items={checklistItems} />
```

Comment:
I believe SecurityCheckList also needs two copy variants, please separate it too 🙏

Reply:
No alternative copy was provided, can we merge this and wait for @Hermez-cz with any additional changes?

Reply:
Ah, I missed that. Adding 7ac0b57f17b0c40dd2d34a92bee22e3c5c221290 as a separate commit in case we need to revert it as I don't like this approach. Crowdin has no power over the icons, number of items or translation variables and we are not even sure if all of the items will change.

Reply:
I see, it's not completely dynamic, but it didn't cost too much effort and it gives a chance to play with copy a bit, so I think it's worth it... 👍